WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewViewModel.swift

```diff
@@ -7,12 +7,29 @@ import protocol Storage.StorageManagerType
 /// View model for `ProductDetailPreviewView`
 ///
 final class ProductDetailPreviewViewModel: ObservableObject {
+    enum ProductDetailField: Equatable {
+        case name
+        case shortDescription
+        case description
+    }
+
+    struct NameSummaryDescOption {
+        let name: String
+        let shortDescription: String
+        let description: String
+    }
+
     typealias ImageState = EditableImageViewState
 
     @Published private(set) var imageState: ImageState
     @Published private(set) var isGeneratingDetails: Bool = false
     @Published private(set) var isSavingProduct: Bool = false
-    @Published private(set) var generatedProduct: Product?
+    @Published private(set) var generatedAIProduct: AIProduct?
+
+    @Published private(set) var selectedOptionIndex: Int = 0
```

Comment:
Super nit:

```suggestion
    @Published private(set) var selectedOptionIndex = 0
```

Reply:
Done in 2b65f09d5fd0c101c3519b8667bbcaf1ad4475e6